api/pkg/cli/cmd/info/info.go

```diff
@@ -172,11 +205,79 @@ func (opts *options) run() error {
 	return printer.New(out).Tabbed(tmpl, tmplData)
 }
 
+func (opts *options) AskCatalogName() (string, error) {
+	// Get all Catalogs
+	catalog, err := opts.hubClient.GetCatalogsList()
+	if err != nil {
+		return "", err
+	}
+	if len(catalog) == 1 {
+		return catalog[0], nil
+	} else {
+		// Ask the catalog
+		err = opts.selectOption.Ask("catalog", catalog)
+		if err != nil {
+			return "", err
+		}
+		return opts.selectOption.Catalog, nil
+	}
+}
+
+func (opts *options) AskResourceName() (string, error) {
+	// Get all resources from the Catalog selected
+	resources, err := opts.hubClient.GetResourcesList(hub.SearchOption{
+		Kinds:   []string{opts.kind},
+		Catalog: opts.from,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	// Ask the resource name
+	err = opts.selectOption.Ask(opts.kind, resources)
+	if err != nil {
+		return "", err
+	}
+	return opts.selectOption.Name, nil
+}
+
+func (opts *options) AskVersion(name string) (string, error) {
+	// Get all the versions of the resource selected
+	ver, err := opts.hubClient.GetResourceVersionslist(hub.ResourceOption{
+		Name:    name,
+		Kind:    opts.kind,
+		Catalog: opts.from,
+	})
+	if err != nil {
+		return "", err
+	}
+	if len(ver) == 1 {
```

Comment:
handle if no version

Reply:
If there is no version of the resource then it won't be a hub resource

Reply:
i.e. while parsing the catalog I guess the resource won't be added in the db